### DISCUSSION TITLE
Allow to set the mouse cursor for menu items in the Mega Menu widget

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21830] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

If Open On is set to hover in the megamenu widget, the corser will not be a pointer in the menu item. This gives a very bad feeling to users. But if Open On is set on click, corser will be Pointer.

### Describe the solution you'd like

Please set corser in menu item equal to pointer. Please note that if we set Open On to Hover, the menu item will not be a pointer.

We have two solutions to solve this issue.

1- By default, set corser to pointer.

2- Add an option to control it. This option will have two types of corser. Normal and pointer

![a](https://user-images.githubusercontent.com/88034902/228691707-085657ba-783c-4a9a-8fcd-69c1ab78612e.png)

**Comment by nicholaszein** [comment]
Hello @Michaelgimii,

the reason why the cursor is not a pointer by default is because it should only show if there is a link in the Menu Item. If you don't add a link, it won't show the pointer.

However, I can see the benefit of adding this feature depending on the use case. So thanks for sharing your suggestion! 🙏 

We will consider it in the future if there is enough demand (votes) from other users. 

Kind regards
